rtcdtlstransport.go

```diff
@@ -181,7 +181,12 @@ func (t *RTCDtlsTransport) Start(remoteParameters RTCDtlsParameters) error {
 	// TODO: handle multiple certs
 	cert := t.certificates[0]
 
-	dtlsCofig := &dtls.Config{Certificate: cert.x509Cert, PrivateKey: cert.privateKey, SRTPProtectionProfiles: []dtls.SRTPProtectionProfile{dtls.SRTP_AES128_CM_HMAC_SHA1_80}}
+	dtlsCofig := &dtls.Config{
+		Certificate:            cert.x509Cert,
+		PrivateKey:             cert.privateKey,
+		SRTPProtectionProfiles: []dtls.SRTPProtectionProfile{dtls.SRTP_AES128_CM_HMAC_SHA1_80},
+		ClientAuth:             dtls.RequireAnyClientCert,
```

Comment:
Does this always require client certs? Is that supposed to be a requirement?

Reply:
Yep it is a requirement!

The peers should always validate the certificate using the fingerprint in the SDP